frontend/src/metabase/parameters/components/ParameterValueWidget.jsx

```diff
@@ -135,7 +135,12 @@ class ParameterValueWidget extends Component {
     }
 
     if (!hasNoPopover(this.props.parameter)) {
-      return <WidgetStatusIcon name="chevrondown" />;
+      return (
+        <WidgetStatusIcon
+          name="chevrondown"
+          size={this.props.mimicMantine ? 16 : undefined}
```

Comment:
this is needed to make old style widget to look like new selects from metabase/ui. 

![Uploading Screenshot 2024-04-11 at 23.42.22.png…]()